scripts/test/run_playmode_tests.ps1

```diff
@@ -15,7 +15,11 @@ param(
     # Path to your Unity Executable
     [ValidateScript({[System.IO.File]::Exists($_) -and $_.EndsWith(".exe") })]
     [string]
-    $unityExePath = "C:\Program Files\Unity\Hub\Editor\2018.4.6f1\Editor\Unity.exe"
+    $unityExePath = "C:\Program Files\Unity\Hub\Editor\2018.4.6f1\Editor\Unity.exe",
+    # Optional test filter

```

Comment:
Consider adding one more line of comment like:
# Example: -editorTestsFilter InputSystem
# This will filter to only tests that contain the word "InputSystem"

The reason I say this is because usually the first thing someone wonders is the format, and having an example is really helpful (they can then just copy/paste/substitute)

Reply:
Wait... that's two lines of comments...

Reply:
😱😱